docs/operation/operation.md

```diff
@@ -398,3 +413,53 @@ In order to control this limits, there are two parameters: `limit` and
 to get the results paginated or getting all of them at the same time.
 
     % curl localhost:9911/routes?offset=200&limit=100
+
+# Memory consumption
+
+Memory consumption is in general no problem, but some features can if
```

Comment:
"Memory consumption is in general no problem, but some features can if
not carefully used create more memory consumption that you might
thought.

Heavy memory consumers:
- Metrics
- Filters
- Slow Backends and chatty clients"

could be:

"While Skipper is generally not memory bound, some features may require
some attention and planning regarding the memory consumption.

Potentially high memory consumers:
- Metrics
- Filters
- Slow Backends and chatty clients"